libp2p/standard_setup.nim

```diff
@@ -1,11 +1,20 @@
+# compile time options here
+const
+  libp2p_secure {.strdefine.} = ""
```

Comment:
can this be negotiated / dynamic?

Reply:
Well, like @dryajov said in the call yesterday:
1. multistream select as it stands has no ordering warranty
2. for NBC noise should be the only one allowed
So we decided to use compile time flags.

Indeed for other generic usages it should be just added to the table in the same file to be an option of the 2 between secio and noise.

Reply:
@arnetheduck it is negotiated dynamically but the order of why one is chosen isn't guaranteed, so we want to always run with either secio or noise for now.